src/yoroi-wallets/walletManager.ts

```diff
@@ -539,22 +517,13 @@ export class WalletManager {
     await this.closeWallet()
     await storage.remove(`/wallet/${id}/data`)
     await storage.remove(`/wallet/${id}`)
-
-    this._wallets = _.omit(this._wallets, id)
   }
 
   // TODO(ppershing): how should we deal with race conditions?
```

Comment:
is this comment still relevant after your changes?

Reply:
I don't know